dev-python/pycuda/pycuda-2018.1.1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
This copyright mark is old, you should use the latest repoman to commit to fix it automatically.

Reply:
done